cmd/kubeadm/app/util/users/users_linux.go

```diff
@@ -629,3 +631,26 @@ func writeFile(f *os.File, str string) error {
 	}
 	return nil
 }
+
+// UpdateFileOwnerAndPermissions updates a files owner and permissions.
+func UpdateFileOwnerAndPermissions(path string, uid, gid int64, perms uint32) error {
+	if err := os.Chown(path, int(uid), int(gid)); err != nil {
+		return err
+	}
+	if err := os.Chmod(path, fs.FileMode(perms)); err != nil {
+		return err
```

Comment:
better to have:
`errors.Wrap(err, some description)`

Reply:
Done!